coderd/telemetry/telemetry.go

```diff
@@ -473,13 +474,24 @@ func (r *remoteReporter) createSnapshot() (*Snapshot, error) {
 		return nil
 	})
 	eg.Go(func() error {
-		stats, err := r.options.Database.GetWorkspaceAgentStats(ctx, createdAfter)
-		if err != nil {
-			return xerrors.Errorf("get workspace agent stats: %w", err)
-		}
-		snapshot.WorkspaceAgentStats = make([]WorkspaceAgentStat, 0, len(stats))
-		for _, stat := range stats {
-			snapshot.WorkspaceAgentStats = append(snapshot.WorkspaceAgentStats, ConvertWorkspaceAgentStat(stat))
+		if r.options.DeploymentConfig != nil && slices.Contains(r.options.DeploymentConfig.Experiments, string(codersdk.ExperimentWorkspaceUsage)) {
```

Comment:
Like a few other things in this fix, it's a bit hacky, but this is fine if we clean it out when the experiment is over.